nexus/types/src/deployment.rs

```diff
@@ -335,6 +335,11 @@ pub struct BlueprintZonesConfig {
 }
 
 impl BlueprintZonesConfig {
+    /// Constructs an empty [`BlueprintZonesConfig`]
+    pub fn empty() -> Self {
+        BlueprintZonesConfig { generation: Generation::new(), zones: vec![] }
+    }
```

Comment:
Can this be the `Default` impl for `BlueprintZonesConfig`?

Reply:
It could.  I usually prefer more explicit, named methods because `default()` doesn't convey semantics (this would not always be an appropriate default value -- really only if you know the corresponding sled has no zones).  But I get it's more idiomatic and I can do that.